src/internal/operators/OperatorSubscriber.ts

```diff
@@ -1,6 +1,16 @@
 /** @prettier */
 import { Subscriber } from '../Subscriber';
 
+// Ignore TypeScript's errors about the following methods being protected.
+// They're stored here to avoid their needing to be looked up in the
+// constructor.
+// @ts-ignore
+const defaultOnNext = Subscriber.prototype._next;
```

Comment:
I think you can just use `super._next` below. :)

Reply:
`super` cannot be used in a parameter declaration; it can only be used after `super()` has been called.

Reply:
I really shouldn't correspond until I've had m morning coffee. A previous iteration to what's in this commit was using default parameter and I forgot that I'd changed it. 😅 Sorry.

As it is, do we really want to use `super` with a property lookup just to appease two build-time tools? TypeScript  - because of the `protected` business - and the `check-side-effects` - which freaks out when it sees property accesses. Why pay a runtime cost - however small - to make those tools happy? As it is, this class is already doing weird stuff by overloading prototype methods with instance properties.

Reply:
🤷‍♂️ The other way, there's an additional closure which is also a cost. All I know it's it's what `super` is for. Seems like we should use it there. If it's really down to performance, we should be profiling this to make the decision, I guess.

Reply:
I don't want to be argumentative, but where is the additional closure?